scipy/optimize/_linprog_ip.py

```diff
@@ -794,7 +794,8 @@ def eta(g=gamma):
         go = rho_p > tol or rho_d > tol or rho_A > tol
 
         if disp:
-            _display_iter(rho_p, rho_d, rho_g, alpha, float(rho_mu), obj)
+            _display_iter(rho_p, rho_d, rho_g, float(alpha),
```

Comment:
Minor style nitpick, I think it would be more readable to convert this inside `` _display_iter ``. Casting to a float is conceptually a part of ``_display_iter`` (otherwise it would not be needed). Further, someone who is reading _linprog_ip won't have to think about **why** these are being cast - instead only seeing it when it becomes relevant.

Reply:
Fair. And I should probably cast them all.

Reply:
Done.